modules/swagger-codegen/src/main/resources/cpprest/model-source.mustache

```diff
@@ -251,13 +286,15 @@ void {{classname}}::fromJson(web::json::value& val)
         {{/isDateTime}}
         {{^isDateTime}}
         if(!val[utility::conversions::to_string_t("{{baseName}}")].is_null())
-        {
-            {{{datatype}}} newItem({{{defaultValue}}});
-            newItem->fromJson(val[utility::conversions::to_string_t("{{baseName}}")]);
-            {{setter}}( newItem );
+        {        	
+           	{{{datatype}}} newItem(getNew{{nameInCamelCase}}Instance());
```

Comment:
this will produce memory leaks

Reply:
I don't see why but I changed this part anyway so it (hopefully) is okay with you.